Extension/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.21.1: July 9, 2024
+### Enhancements
+* Add a `default` value for the `C_Cpp.experimentalFeatures` setting (enabling the `disabled` value to work). [#12163](https://github.com/microsoft/vscode-cpptools/issues/12163)
```

Comment:
This one has not yet been approved. We need to discuss it more.

Reply:
@bobbrow Yeah, I just closed that PR unless we have a new experiment in the future.